README.md

```diff
@@ -318,11 +322,17 @@ TestFlight.allInfo()
 
 Please read [CONTRIBUTING.md](https://github.com/giovaniceotto/RocketPy/blob/master/CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us. - **_Still working on this!_**
 
-## Authors
+## Authors and Contributors
+
+This package was originally created by [Giovani Ceotto](https://github.com/giovaniceotto/) as part of his work at Projeto Jupiter. [Rodrigo Schmitt](https://github.com/rodrigo-schmitt/) was one of the first contributors.
+
+Later, [Guilherme Fernandes](https://github.com/Gui-FernandesBR/) and [Lucas Azevedo](https://github.com/lucasfourier/) joined the team to work on the expansion and sustainability of this project.
+
+Since then, the team has been growing fast and our contributors are what makes us special!
```

Comment:
What about citing @Projeto-Jupiter/rocketpy-team as well?  Would it be "too much"?

Reply:
Great idea!